src/components/Header_type_1.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import { routeTo } from 'redux-router-kit';
+import { getTranslator } from '../reducers/translation';
+import { RoundHeaderButton } from './Buttons';
+
+import styles from './Header_type_1.scss';
+import common from '../styles/common.scss';
+
+const HeaderInitialComponent = ({ i18n, targetTo }) =>
+  (<header>
+    <div className={common.widthLimiter}>
+      <div className={styles.header}>
+        <Grid fluid>
+          <Row>
+            <Col lg={1} className={styles.logoContainer}>
+              <div className={styles.logo} />
+            </Col>
+            <Col lg={8} className={styles.mainArea}>
+              <div className={styles.title}>
+                {i18n('Neufund Platform')}
+              </div>
+              <div className={styles.status}>
+                {i18n('Log in')}
+              </div>
+            </Col>
+            <Col lg={3} className={styles.createButton}>
+              <RoundHeaderButton
+                onClick={(event) => {
+                  event.preventDefault();
```

Comment:
Do we need `preventDefault` here? I thought it was only required for `<a>` tags. (And then `Link` should take care of it for us.)